tests/components/insteon/test_config_flow.py

```diff
@@ -648,3 +651,48 @@ async def test_discovery_via_usb_already_setup(hass):
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "single_instance_allowed"
+
+
+async def test_discovery_via_dhcp_hubv1(hass):
+    """Test usb flow."""
+    await _test_dhcp(hass, HUB1)
+
+
+async def test_discovery_via_dhcp_hubv2(hass):
+    """Test usb flow."""
+    await _test_dhcp(hass, HUB2)
```

Comment:
We can use pytest.mark.parametrize to parametrize the test instead.

https://docs.pytest.org/en/latest/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions